src/messages/components/ConversationCompose.vue

```diff
@@ -125,17 +136,28 @@ export default {
     },
   },
   methods: {
+    getKey () {
+      if (this.key) {
+        return this.key
+      }
+      else return 'wallConv'
+    },
     submit () {
       this.$emit('submit', this.message)
+      localStorage.removeItem(this.getKey())
     },
     leaveEdit () {
       this.$emit('leaveEdit')
+      this.message = ''
+      localStorage.removeItem(this.getkey())
```

Comment:
There's a small bug in this: it would delete a message draft from localStorage when editing a previous message in the same conversation. 

For example, I start writing a new message to you, then edit an old message in our conversation but abort editing and close my browser. I would expect the new message to still exist when I open the browser again.

Reply:
on the wall there is a cancel button that user can use to cancel the editing and then accept the draft to be deleted but on pickup messages the only way to cancel is to press the escape button. I think it would make sense to add a button here and in such case, I agree this deletion is not necessary. but without this button, i guess most users will just leave the draft thinking that it would cancel the draft. It is more a UX question

Reply:
I'm not sure what you are referring to: as far as I know, the "cancel edit" button shown in both WallConversation and ChatConversation.